### PR TITLE
Setting Up Devise

### PR DESCRIPTION
**Pull request by ridwvn22**
### Changes
Give a short summary of the changes made in this PR. 
Generated devise, views, and added a partial.

### Link to Issue 
Link Asana ticket here!
https://app.asana.com/0/1206510424537042/1206673931535427/f

### Comments, Notes, or  Questions
This section is if you want to share something related to the PR with the reviewer. 
Pair Programmed with Creed to set up devise User model

### Related Screenshots 
For FE tickets specifcially, please include a screenshot of your work!

### Before you submit for review, Did you...

- [ ] Added relevant tests (if applicable)
- [X ] Linked Asana ticket to PR
- [X ] Added comments, notes, or questions for reviewwer
- [ ] Added relevant screenshots
- [X] Ensured ticket title is clear and concise(i.e. 'Donation Form Feature' instead of 'sfg/chels/donation-form'

### CODE REVIEW

**Review by edithemmings — CHANGES_REQUESTED** [review]
Please make sure that tests and lint actions are passing before requesting review

File: README.md [hunk 1]
```diff
@@ -12,15 +12,15 @@ HAPPY CODING!
 For this cohort, you will be working on the educator user of an application called 'Innocent Tech.'
 Innocent Tech is a full-stack application that works to eradicate racial bias in the classroom to better support students of color. 
 There will be an Innocent Tech kick off the first week of the apprenticeship.
-- [ ] I received and accepted the 'Innocent Tech Kickoff' invite
+- [X] I received and accepted the 'Innocent Tech Kickoff' invite
```
Comment (edithemmings) [inline README.md:15]: It looks like you branched from your readme work. Please branch rebase with the `main` branch

File: package.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
+    "tailwindcss": "^3.4.1"
```
Comment (edithemmings) [inline package.json:3]: Tailwind was added as a rails gem, so I don't think we need it the npm package. @camckin10 is there any reason we'd need both?

File: app/views/layouts/application.html.erb [hunk 3]
```diff
@@ -12,6 +12,11 @@
   <body>
     <main class="container mx-auto mt-28 px-5 flex">
       <!--This is where you will add render lines to render the partials-->
+      <% if current_user %>
+      <%= render 'layouts/sidebar_nav' %>
```
Comment (edithemmings) [inline app/views/layouts/application.html.erb:16]: This looks like it needs one more tab to the right

File: app/views/layouts/application.html.erb [hunk 4]
```diff
@@ -12,6 +12,11 @@
...
+      <% end %> 
+      <p class="notice"><%= notice %></p>
+      <p class="alert"><%= alert %></p>
```
Comment (edithemmings) [inline app/views/layouts/application.html.erb:19]: What is the reason for adding notice and alert?

File: config/routes.rb [hunk 5]
```diff
@@ -1,13 +1,15 @@
...
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
   # Reveal health status on /up that returns 200 if the app boots with no exceptions, otherwise 500.
   # Can be used by load balancers and uptime monitors to verify that the app is live.
   # get 'up' => 'rails/health#show', as: :rails_health_check
 
   # Defines the root path route ("/")
-  # root "posts#index"
-  root 'application#index'
+  root "home#index"
```
Comment (edithemmings) [inline config/routes.rb:12]: This should stay as `application`

File: app/views/layouts/application.html.erb (hunk 4 above)
Comment (ridwvn22) [reply app/views/layouts/application.html.erb:19]: It was one of the required steps after running rails generate devise:install 
3. Ensure you have flash messages in app/views/layouts/application.html.erb.
     For example:

       <p class="notice"><%= notice %></p>
       <p class="alert"><%= alert %></p>

**Review by edithemmings — COMMENTED** [review]
We just connected on a call and the 'rebase' process required creating a different branch. So this PR will be closed out.

**Pull request closed without merging**